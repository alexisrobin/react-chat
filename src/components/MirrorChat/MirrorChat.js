import React from 'react';
import { Container, Row, Col } from 'reactstrap';
import Chat from '../Chat';

const MirrorChat = ({ messages, users, addMessage }) => {
    return (
        <Container>
            <Row>
                <Col md={6}>
                    <Chat
                        messages={messages}
                        users={users}
                        addMessage={addMessage}
                        currentUserId={1} />
                </Col>
                <Col md={6}>
                    <Chat
                        messages={messages}
                        users={users}
                        addMessage={addMessage}
                        currentUserId={2} />
                </Col>
            </Row>
        </Container>
    )
}

export default MirrorChat;