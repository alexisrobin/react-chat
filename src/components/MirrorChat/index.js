import MirrorChat from './MirrorChat';
export default MirrorChat;