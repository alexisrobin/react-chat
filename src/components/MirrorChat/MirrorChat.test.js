import React from 'react';
import { expect } from 'chai';
import { shallow } from 'enzyme';
import MirrorChat from './MirrorChat';

describe('<MirrorChat />', () => {
    it('should match its reference snapshot', () => {
        const wrapper = shallow(
            <MirrorChat />
        );

        expect(wrapper).to.matchSnapshot()
    })
})