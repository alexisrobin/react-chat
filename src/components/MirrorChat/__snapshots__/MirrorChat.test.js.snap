// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MirrorChat /> should match its reference snapshot 1`] = `
<Container
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      md={6}
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Chat
        currentUserId={1}
      />
    </Col>
    <Col
      md={6}
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Chat
        currentUserId={2}
      />
    </Col>
  </Row>
</Container>
`;
